build.py

```diff
@@ -65,4 +65,4 @@ def build(package, version, optimization=0, out_dir=os.path.join(os.getcwd(),
 
 if __name__ == '__main__':
     args = parser.parse_args()
-    build(args.package, args.version)
+    build(args.package, args.version, args.optimization)
```

Comment:
That's very suspicious, looks like that the test binaries weren't generated by this script